doc/release-notes/release-8.sgml.in

```diff
@@ -33,6 +33,16 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 	<item>
 </itemize>
 
+<sect1>Helper changes
+<p>
+<descrip>
+	<p>Removed <em>ext_lm_group_acl</em> external acl helper. No product
+       currently on the market suports the LanManager protocol
+       in their default configuration.
```

Comment:
Release notes indentation is arguably inconsistent already, but please do not use spaces for primary (i.e. left-most) indentation of lines in indented `<p>` paragraphs (in this PR) because official code uses tabs for that purpose:

```
$ grep -A1 '.<p>' doc/release-notes/release-7.sgml.in | grep -c '^ '
0
```

There are two places in this PR that add inconsistent indentation.

I am aware of recently fixed CI formatting checks failures.

Reply:
FTR; I think that was github editor changing the indents on my notes update.